cirq/iqm/iqm_device.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""IQM builds quantum computers based on superconducting transmon technology.
+For more information, see https://meetiqm.com.
```

Comment:
Consider putting this in the __init__.py since it applies to all devices in iqm/ (e.g. if a second one is added later)